core/src/main/java/io/questdb/cairo/sql/PartitionFrameCursor.java

```diff
@@ -44,8 +44,11 @@ default void calculateSize(RecordCursor.Counter counter) {
      * @return the next element in the partition frame
      * @throws io.questdb.cairo.DataUnavailableException when the queried partition is in cold storage
      */
-    @Nullable
-    PartitionFrame next();
+    default @Nullable PartitionFrame next() {
+        return next(0);
+    }
+
+    @Nullable PartitionFrame next(long skipTarget);
```

Comment:
nit: would be great to document this method

Reply:
added javadoc here: https://github.com/questdb/questdb/pull/5939/commits/4091a1cba172650cb482cbd097e50a5784658b27